services/device-registry-mongodb/src/main/java/org/eclipse/hono/deviceregistry/mongodb/utils/MongoDbDocumentBuilder.java

```diff
@@ -27,6 +27,10 @@
  */
 public final class MongoDbDocumentBuilder {
 
+    public static final String TENANT_TRUSTED_CA_SUBJECT_PATH = String.format("%s.%s.%s",
```

Comment:
This field is used only within this class and it should be private.

Reply:
done